Backpack/src/main/java/net/skyscanner/backpack/gradient/BpkGradients.kt

```diff
@@ -18,7 +18,7 @@ import net.skyscanner.backpack.util.resolveThemeColor
 open class BpkGradients @JvmOverloads constructor(
   context: Context,
   orientation: Orientation = Orientation.TL_BR,
-  colors: IntArray = intArrayOf(ContextCompat.getColor(context, R.color.bpkSkyBlue), ContextCompat.getColor(context, R.color.bpkPrimaryGradientLight))
+  colors: IntArray = intArrayOf(ContextCompat.getColor(context, R.color.bpkSkyBlue), ContextCompat.getColor(context, R.color.bpkSkyBlue))
```

Comment:
Can we not change this? I understand the script is looking for `Light` and `Dark` now to put the colours in the right folder but we should ignore this